indico_zodbimport/modules/event_papers.py

```diff
@@ -55,21 +83,45 @@ def _translate_notif_options(pr, options):
     return {PaperReviewingRole[role] for role, (attr, default) in options.viewitems() if getattr(pr, attr, default)}
 
 
+def _review_color(review, text):
+    return '%{{{}}}{}%{{reset}}'.format({
```

Comment:
I usually go for `return '%%{%s}{}%%{reset}' % (...)` - but it's only slightly less ugly